src/include/sof/notifier.h

```diff
@@ -37,29 +37,31 @@
 
 struct sof;
 
-/* notifier general IDs */
-#define NOTIFIER_ID_CPU_FREQ	0
-#define NOTIFIER_ID_SSP_FREQ	1
-
 /* notifier target core masks */
 #define NOTIFIER_TARGET_CORE_MASK(x)	(1 << x)
 #define NOTIFIER_TARGET_CORE_ALL_MASK	0xFFFFFFFF
 
+enum notify_id {
+	NOTIFIER_ID_CPU_FREQ = 0,
+	NOTIFIER_ID_SSP_FREQ,
+	NOTIFIER_ID_KEY_PHRASE_DETECTED,
```

Comment:
Pls replace with some more generic name for all events sent by KPB clients, e.g. `NOTIFIER_ID_KPB_CLIENT_EVT'